cloudferrylib/os/actions/transport_storage_resources.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2014 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import log
+from cloudferrylib.utils import utils as utl
+
+
+LOG = log.getLogger(__name__)
+
+
+class TransportStorageResources(action.Action):
```

Comment:
Please add documentation in [sphinx format](http://www.sphinx-doc.org/en/stable/contents.html) describing:
 - High-level overview of the action (what it does);
 - Required configuration options
 - Other actions which depend on this action
 - Actions this action depends on